setup.cfg

```diff
@@ -1,17 +1,17 @@
 [flake8]
 max-line-length = 120
-exclude = docs/src, build, dist
+exclude = docs/src, build, dist, .ipynb_checkpoints
 
 [isort]
 line_length = 120
 not_skip = __init__.py
+skip_glob = .ipynb_checkpoints
 known_first_party = pyro, tests
 known_third_party = opt_einsum, six, torch, torchvision
 
 [tool:pytest]
 filterwarnings = error
     ignore::DeprecationWarning
-    once::DeprecationWarning
```

Comment:
@fehiepsi Can you please revert this change? It is unrelated to the PR title and breaks previously tuned logging. These kind of tiny "cleanup" tweaks are not necessary. Let's spend time on useful things like GPyTorch integration.

Reply:
@fritzo Though it might not be necessary, these works really help me relax my mind after reading hard-to-understand papers. I just do this clean-up job periodically for that reason.

Reply:
+1. Both `once` and `ignore` would be ignored by pytest. `once` will print the warning only for the first occurrence of such a warning, and `ignore` would not print the warning at all. The reason why it was set to `once` is so that we see warnings at the end of pytest summary about deprecated methods in external dependencies, and can take appropriate action ahead of time. Let us not remove this! :)

Reply:
It seems complicated. Haizzz....

Reply:
@fehiepsi understood, I also find it relaxing to clean up after reading papers. Just remember that cleaning up Pyro also incurs review overhead, and reviewing subtle config PRs may not be relaxing to reviewers :wink: